integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -160,6 +162,88 @@ def _check_matched_content_updated_at(
                 items_to_update[matched_item.get('content_key')] = item
         return items_to_update
 
+    def _check_matched_content_to_create(
+        self,
+        enterprise_customer_catalog,
+        matched_items
+    ):
+        """
+        Take a list of content keys and create ContentMetadataItemTransmission records. When existed soft-deleted
+        records exist, resurrect them. When created but not-yet-transmitted records exist, include them.
+
+        Args:
+            enterprise_customer_catalog (EnterpriseCustomerCatalog): The enterprise catalog object
+
+            matched_items (list): A list of dicts containing content keys and the last datetime that the respective
+                content was updated
+
+            force_retrieve_all_catalogs (Bool): If set to True, all content under the catalog will be retrieved,
+                regardless of the last updated at time
+        """
+        ContentMetadataItemTransmission = apps.get_model(
+            'integrated_channel',
+            'ContentMetadataItemTransmission'
+        )
+        items_to_create = {}
+        create_transmissions = []
+        for matched_item in matched_items:
+            content_id = matched_item.get('content_key')
+            content_last_changed = matched_item.get('date_updated')
+            past_deleted_transmission = ContentMetadataItemTransmission.objects.filter(
```

Comment:
nit: could create a convenience classmethod on `ContentMetadataItemTransmission` that takes only `self.enterprise_configuration` and `content_id` as arguments.